src/frontend/components/Felleskomponenter/Datovelger/Datovelger.tsx

```diff
@@ -56,53 +43,79 @@ const Datovelger: React.FC<DatoVelgerProps> = ({
     label,
     disabled = false,
     dynamisk = false,
-    calendarPosition = '',
+    strategy = 'fixed',
 }) => {
     const [valgtLocale] = useSprakContext();
     const { tekster, plainTekst } = useApp();
     const { datoformatHjelpetekst, datoformatPlaceholder } = tekster().FELLES.hjelpeteksterForInput;
 
-    const hentBegrensninger = () => {
-        const limitations: DatepickerLimitations = {};
+    const minDatoErIFremtiden = () =>
+        tilhørendeFraOgMedFelt?.verdi &&
+        hentFromDate() !== undefined &&
+        isAfter(hentFromDate() as Date, dagensDato());
+
+    const hentFromDate = (): Date | undefined => {
+        let minDato = tidenesMorgen();
 
         if (avgrensMinDato) {
-            limitations.minDate = avgrensMinDato;
-        } else if (tilhørendeFraOgMedFelt) {
-            limitations.minDate = dayjs(tilhørendeFraOgMedFelt.verdi)
-                .add(1, 'day')
-                .format('YYYY-MM-DD');
+            minDato = avgrensMinDato;
+        } else if (tilhørendeFraOgMedFelt?.verdi) {
+            minDato = dagenEtterDato(stringTilDate(tilhørendeFraOgMedFelt.verdi));
         }
+        return minDato;
+    };
+
+    const hentToDate = (): Date => {
+        let maxDato = tidenesEnde();
 
         if (avgrensDatoFremITid || avgrensMaxDato) {
-            limitations.maxDate = avgrensMaxDato ? avgrensMaxDato : dagensDato();
+            maxDato = avgrensMaxDato ? avgrensMaxDato : dagensDato();
         }
 
-        return limitations;
+        return maxDato;
     };
 
+    const { datepickerProps, inputProps, reset } = UNSAFE_useDatepicker({
+        locale: valgtLocale,
+        fromDate: hentFromDate(),
+        toDate: hentToDate(),
+        today: minDatoErIFremtiden() ? hentFromDate() : dagensDato(),
+        defaultSelected: parseTilGyldigDato(felt.verdi, 'yyyy-MM-dd'),
+        onDateChange: (dato: Date | undefined) => {
+            if (dato) {
+                felt.validerOgSettFelt(formatISO(dato), { representation: 'date' });
```

Comment:
Her er det litt kluss med paranteser som gjør at datoen ikke blir på formatet yyyy-MM-dd. Ved å fikse denne burde det ikke være nødvendig med `useEffect`-hooket under her 👇 
```suggestion
                felt.validerOgSettFelt(formatISO(dato, { representation: 'date' }));
```